Automattic/portkey-android
# Story creation: allow multiselect and add multiple frames (1)

Fix #245 

Builds on top of #241 

This PR implements multiple selection in the media picker, and prepends all selected items in the frame array for the current Story.

To test:
1. open the app and tap + to enter the capture mode
2. tap on the Upload icon to enter the photo picker
3. select several videos / images
4. tap on Next or on the ✔️ on the action bar
5. observe all items are added as frames to your Story (see bottom strip)

![multiselect](https://user-images.githubusercontent.com/6597771/72107006-3da57100-330f-11ea-9463-d16a00bbd85f.gif)
